src/stories/Library/Modals/modal-profile/modal-profile.scss

```diff
@@ -34,3 +63,7 @@
     }
   }
 }
+
+.test-color {
```

Comment:
Should this stay here?

Reply:
Absolutely not.. Thanks for pointing it out.